pkg/app/master/commands/xray/handler.go

```diff
@@ -1268,18 +1268,30 @@ func printObject(xc *app.ExecutionContext, object *dockerimage.ObjectMetadata) {
 	if object.Hash != "" {
 		hashInfo = fmt.Sprintf(" hash=%s", object.Hash)
 	}
-
-	fmt.Printf("%s: mode=%s size.human='%v' size.bytes=%d uid=%d gid=%d mtime='%s' %s%s '%s'",
-		object.Change,
-		object.Mode,
-		humanize.Bytes(uint64(object.Size)),
-		object.Size,
-		object.UID,
-		object.GID,
-		object.ModTime.UTC().Format(time.RFC3339),
-		objectHistoryString(object.History),
-		hashInfo,
-		object.Name)
+	xc.Out.Info("params",
+		ovars{
+			"mode":        object.Mode,
+			"size.human":  humanize.Bytes(uint64(object.Size)),
+			"size.bytes":  object.Size,
+			"uid":         object.UID,
+			"gid":         object.GID,
+			"mtime":       object.ModTime.UTC().Format(time.RFC3339),
+			"H":           objectHistoryString(object.History),
+			"hash":        hashInfo,
+			"object.name": object.Name,
+		},
+	)
+	// fmt.Printf("%s: mode=%s size.human='%v' size.bytes=%d uid=%d gid=%d mtime='%s' %s%s '%s'",
+	// 	object.Change,
+	// 	object.Mode,
+	// 	humanize.Bytes(uint64(object.Size)),
+	// 	object.Size,
+	// 	object.UID,
+	// 	object.GID,
+	// 	object.ModTime.UTC().Format(time.RFC3339),
+	// 	objectHistoryString(object.History),
+	// 	hashInfo,
+	// 	object.Name)
 
 	if object.LinkTarget != "" {
```

Comment:
also need to include this print statement in the json-formatted version

Reply:
`object.LinkTarget` right? I had a doubt here actually, in the case its empty, should we use `\n` or just `""` will do?